src/content/getting-started/developers/other-frameworks.mdx

```diff
@@ -3,15 +3,31 @@ title: Developers
 tabs: ['Vanilla', 'React', 'Angular', 'Other frameworks']
 ---
 
+<AnchorLinks>
+
+- [Development options](#development-options)
+- [Other frameworks?](#other-frameworks)
+
+</AnchorLinks>
+
 ## Development options
 
+<AnchorLinks small>
+
+- [Using just the styles](#using-just-the-styles)
+- [Wrapping a component with a JavaScript framework of your choice](#wrapping-a-component-with-a-javascript-framework-of-your-choice)
+- [Examples](#examples)
+- [Troubleshooting](#troubleshooting)
+
+</AnchorLinks>
+
 The Carbon Design System supports vanilla JS, React and Angular as core parts of the product. But you can still build components even if you're using a different framework.
 
-**Using just the styles**
+### Using just the styles
 
 Developers wanting to build in different ways follow the instructions for the [Vanilla](/getting-started/developers/vanilla) library to access the styles and build out their own components.
```

Comment:
is this a behavior we want to encourage? "just using the styles" is a frequently encountered practice that is seem as equivalent to "using Carbon"; unless they would pick up future enhancements, we should be clear when this is OK and when it's Not OK. I assume this is really directed at teams creating their own frameworks of components rather than extending existing frameworks via submission or add-ons.